app/jobs/document_proofing_job.rb

```diff
@@ -40,12 +43,26 @@ def perform(result_id:, encrypted_arguments:, trace_id:,
       end
     end
 
-    dcs = DocumentCaptureSession.new(result_id: result_id)
-
     dcs.store_doc_auth_result(
       result: proofer_result.to_h, # pii_from_doc is excluded from to_h to stop accidental logging
       pii: proofer_result.pii_from_doc,
     )
+
+    analytics = Analytics.new(user: user, request: nil, sp: nil)
```

Comment:
>Can we assign sp from dcs.issuer ? I think we use this detail in some dashboards (cc @amathews-fs).

Yep, thank you, forgot we had `issuer` on that

>Is there a meaningful difference between the analytics instance we create here, vs. the logger we call a few lines below in the ensure block? Should they be the same?

That log goes to `production.log`. I don't think we need this in the ensure since it depends a bit more on the result and other DB calls.